cloud_functions/climateiq_data_export_cf/main.py

```diff
@@ -108,6 +119,7 @@ def _get_study_area_metadata(study_area_name: str) -> dict:
     db = firestore.Client()
 
     study_area_ref = db.collection(STUDY_AREAS_ID).document(study_area_name)
+    chunks_ref = study_area_ref.collection(CHUNKS_ID)
```

Comment:
there are some firestore accessors in here you can use if they seem helpful:
https://github.com/UrbanSystemsLab/climateiq-cnn/blob/main/usl_pipeline/usl_lib/usl_lib/storage/metastore.py

Reply:
ack I have a TODO to potentially uses these accessors later on